Resources/Prototypes/Body/Organs/arachnid.yml

```diff
@@ -52,120 +27,44 @@
     updateInterval: 1.5
 
 - type: entity
+  parent: [BaseOrganLungs, BaseArachnidOrgan]
   id: OrganArachnidLungs
-  parent: BaseArachnidOrgan
-  name: lungs
-  description: "Filters oxygen from an atmosphere... just more greedily."
+  description: Filters oxygen from an atmosphere... just more greedily.
   components:
-  - type: Sprite
-    layers:
-      - state: lung-l
-      - state: lung-r
-  - type: Lung
   - type: Metabolizer
     updateInterval: 1.5
-    removeEmpty: true
-    solutionOnBody: false
-    solution: "Lung"
-    metabolizerTypes: [ Human ]
-    groups:
-    - id: Gas
-      rateModifier: 100.0
-  - type: SolutionContainerManager
-    solutions:
-      organ:
-        reagents:
-        - ReagentId: Nutriment
-          Quantity: 10
-      Lung:
-        maxVol: 100.0
-        canReact: false
-      food:
-        maxVol: 5
-        reagents:
-        - ReagentId: UncookedAnimalProteins
-          Quantity: 5
 
 - type: entity
+  parent: [BaseOrganHeart, BaseArachnidOrgan]
   id: OrganArachnidHeart
-  parent: BaseArachnidOrgan
-  name: heart
-  description: "A disgustingly persistent little biological pump made for spiders."
+  description: A disgustingly persistent little biological pump made for spiders.
   components:
-  - type: Sprite
-    state: heart-on
-  - type: Item
-    size: Small
-    heldPrefix: heart
   - type: Metabolizer
     updateInterval: 1.5
-    maxReagents: 2
     metabolizerTypes: [Arachnid]
-    groups:
-    - id: Medicine
-    - id: Poison
-    - id: Narcotic
 
 - type: entity
   id: OrganArachnidLiver
-  parent: BaseHumanOrgan
-  name: liver
-  description: "Pairing suggestion: chianti and fava beans."
+  parent: [BaseHumanOrgan, OrganAnimalLiver, BaseArachnidOrgan] #TODO: sprite arachnid liver and change parent from BaseHumanOrgan to BaseArachnidOrgan
```

Comment:
Not gonna lie, I... really dislike these triple inheritances. We don't even need anything from BaseArachnidOrgan. I think it would be better to just go with BaseHumanOrgan and BaseOrganLiver, and switch BaseHumanOrgan to BaseArachnidOrgan once there is an arachnid sprite?

Reply:
We shouldn't be importing "base species organ" to another species at all.

Reply:
While we could certainly just rename the current BaseHumanOrgans to BaseOrgan, to be used as the parent for the other species, and then "derive" a new (completely identical) BaseHumanOrgan off of BaseOrgan. 
I don't know if there would be any point to do so, if BaseOrgan would still use, be historical necessity because of how the sprites are located, a reference to Human/organs.rsi ? I suppose it would mean that there is no _direct_ reference to human organs on every species, so for that reason it might be worth it even if it's just a naming convention